packages/ui/libs/ui/core/src/index.ts

```diff
@@ -1,2 +1,5 @@
+export * from './components/icons/IconPopover/IconPopover';
+export * from './components/icons/FilterIcon/FilterIcon';
+export * from './components/icons/device-icon/DeviceIcon';
```

Comment:
Don't use such exports in the root folder. Use folder-by-folder import. Like it was previously